src/sql/plan_cache/ob_plan_cache_value.cpp

```diff
@@ -902,7 +902,7 @@ int ObPlanCacheValue::resolve_insert_multi_values_param(ObPlanCacheCtx &pc_ctx,
                                        must_be_positive_idx,
                                        *raw_param_array,
                                        &temp_obj_params))) {
-      LOG_WARN("failed to resolve parames", K(ret));
+      LOG_WARN("failed to resolve params", K(ret));
```

Comment:
error: patch failed: src/sql/code_generator/ob_dml_cg_service.cpp
error: patch failed: src/sql/ob_select_stmt_printer.cpp
error: patch failed: src/sql/plan_cache/ob_plan_cache_value.cpp

remove the modification of the above three file, there is conflict with internal codebase